Add support for extra audiences in JWT SVID

eminwux opened the pull request:
fixes #212

faisal-memon commented:
Thanks @eminwux for this. Changes look good. Can you update the readme `jwt_svids` section to mention this new conifigurable?

eminwux commented:
Hi @faisal-memon, I updated the readme with the new configurable. Also, I fixed the DCO. Let me know if there is anything else I should correct. Thank you!
